chirp/drivers/kguv980p.py

```diff
@@ -0,0 +1,2583 @@
+# melvin.terechenok@gmail.com
+# modify for KG-UV980P using KG-935G, KG-UV8H, KG-UV920P drivers as resources
+
+# Copyright 2019 Pavel Milanes CO7WT <pavelmc@gmail.com>
```

Comment:
Most of the driver was copied from the existing kguv920pa driver.  It was then modified to support the specific changes needed for the 980p.  If it worked I didn't change it.

Reply:
Okay, the diff between those drivers looks pretty substantial. If it's just some format changes or something, it'd be better to subclass and override only what you need, but it looks to me like a lot was changed (thus justifying it being its own file). Can you confirm?

Reply:
But still, if you could mention that in the comments, I think that'd be good, even if it's just because someone may want to refactor the common bits out of them in the future.

Reply:
I modified the comment to say 
# Copyright 2022 Mel Terechenok <melvin.terechenok@gmail.com>
# KG-UV980P driver is a modified version of the KG-UV920P driver
# using KG-935G, KG-UV8H, KG-UV920P, and KG-UV9D Plus drivers as resources
#
# Based on the work of Pavel Milanes CO7WT <pavelmc@gmail.com>
# and Krystian Struzik <toner_82@tlen.pl>
# who figured out the crypt used and made possible the
# Wuoxun KG-UV8D Plus driver.

Reply:
Cool, that'll be great, thanks.